src/app/organisation/exportusers/get.controller.js

```diff
@@ -0,0 +1,38 @@
+const CookieModel = require('../../../common/models/Cookie.class');
+const orgApi = require('../../../common/services/organisationApi');
+const logger = require('../../../common/utils/logger')(__filename);
+const createArrayCsvStringifier = require('csv-writer').createArrayCsvStringifier;
+
+
+
+const writeUsersAsCSVtoResponse = (res, orgUsers, orgName) => {
+    const csvStringifier = createArrayCsvStringifier({
+        header: ['Id', 'First Name', 'Last Name', 'Email', 'Role', 'State']
+    });
+
+    const orgSlug = orgName.replace(/[^A-z]/, '-')
+    const dateSlug = new Date().toISOString().slice(0,10);
+    res.setHeader('Content-disposition', 'attachment; filename=' + orgSlug + '-users-' + dateSlug + '.csv');
+    res.setHeader('Content-Type', 'text/csv');
+    
+    res.write(csvStringifier.getHeaderString());
+    res.write(csvStringifier.stringifyRecords(orgUsers));
+    res.end();
+}
+
+module.exports = (req, res) => {
+    logger.debug('In organisation export users get controller');
+    const cookie = new CookieModel(req);
+    const errMSg = { message: 'Failed to get orgusers' };
```

Comment:
is errMsg used here? or may be we need to add error handling?

Reply:
errMsg was not used, have removed. we could add error handling, I'm not sure what we'd do given it's a download process rather than a page that is rendered to user. if if there are exceptions they will be logged?